tex/generic/pgf/utilities/pgfkeys.code.tex

```diff
@@ -45,6 +45,17 @@
   \csname @@end\expandafter\endcsname\expandafter\end%
 \fi
 
+\begingroup
+  \edef\pgfkeys@tmpa{\string\expanded}
+  \edef\pgfkeys@tmpb{\meaning\expanded}
```

Comment:
```suggestion
  \edef\pgfkeys@tmpb{\meaning\pgfkeys@expanded}
```
and move the block after the definition of `\pgfkeys@expanded`

Reply:
Yes, that'd be an easy way to fix this.